chapters/stress/index.md

```diff
@@ -0,0 +1,271 @@
+# Jina Cloud Benchmark & Deployment
+
+#### Purpose: Showcase how Jina scales and how you can deploy to AWS
+
+---
+
+#### Table of Contents
```

Comment:
Should this be small c for contents @FionnD ? Is it auto-generated? If so may need to change the C elsewhere in code

Reply:
No it's not autog, it should be a small c